stories/ng2-component-lab/components/search-bar-example.component.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Created by rc2122 on 11/15/2017.
+ */
+import {Component} from "@angular/core";
+
+@Component({
+    selector: "search-bar-example",
+    template: `
+        <sdc-search-bar placeholder="search text" 
```

Comment:
This should be experiment by itself, because in experiment user see only: <search-bar-example></search-bar-example>
and dose not understand how to write search component.